experimental/ModelBuilder/ModelBuilder.cpp

```diff
@@ -29,6 +30,18 @@ using namespace mlir::edsc::intrinsics;
 
 thread_local MLIRContext mlir::ModelBuilder::ctx;
 
+void ModelBuilder::registerAllDialects() {
```

Comment:
This is quite similar to
https://github.com/google/iree/blob/6b7b71bdcf024caf8b302b06a9ee608f9126a997/iree/tools/init_mlir_dialects.h#L40-L56
expect for the missing `OpenMPDialect`. You still might want to consider to use the IREE function and furthermore to drop the include on `mlir/InitAllDialects.h`.

Reply:
Obsolete as dependencies on IREE should be avoided.